### DISCUSSION TITLE
Add monthly stats how much tests completed/failed etc.

### DISCUSSION THREAD

**Discussion by qarmin** [RustPython/RustPython#4148] [Ideas]
I like rust codegen reports which shows how big progress happened in recent month.
https://blog.antoyo.xyz/rustc_codegen_gcc-progress-report-15

E.g.
Test passing
|Month|Python version|Passed|Failed|Ignored|
|-|-|-|-|-|
|February 2022|3.9|2020|1224|1445|
|March 2022|3.9|2025|1222|1441|
|April 2022|3.10|2045|1210|1440|

**Comment by youknowone** [comment]
That's a good idea.
We have a dashboard  https://rustpython.github.io/pages/regression-tests-results.html
And we also have history of the data https://github.com/RustPython/rustpython.github.io/commits/master

We only need aggregated interface of them to achieve it.
